src/desktop/apps/fair/templates/overview.jade

```diff
@@ -13,6 +13,6 @@ block body
   if primarySets
     include ./overview_bottom
   .for-you-container
-  h2.overview-section-heading Browse the fair
+  .overview-section-heading
```

Comment:
This is a strange construction now, but that allowed me to keep the space. For some reason just updating '.css' there did not really work as expected if I removed the header completely.